internal/api/shared/migrate/mapping.cuepart

```diff
@@ -19,12 +19,30 @@ import (
     }
     // mapping table for the calculation attribute (key = grafana unit, value = perses equivalent)
     calc: {
```

Comment:
Adjusted these a bunch to try to account for all the various use cases in old & new grafana.